pkg/exporter/sumologicexporter/sender.go

```diff
@@ -181,7 +181,16 @@ func (s *sender) logToText(record pdata.LogRecord) string {
 // logToJSON converts LogRecord to a json line, returns it and error eventually
 func (s *sender) logToJSON(record pdata.LogRecord) (string, error) {
 	data := s.filter.filterOut(record.Attributes())
-	data.orig.Upsert(logKey, record.Body())
+
+	// Only append the body when it's not empty to prevent sending 'null' log.
```

Comment:
How about encapsulating this `if` statement in a` function like `isEmpty`?

```
if (!isLogBodyEmpty(body)) {
  data.orig.Upsert(longKey, body)
}
```

Reply:
Done (somewhat like proposed :) )